trino/etc/config.properties

```diff
@@ -4,3 +4,13 @@ node-scheduler.include-coordinator=true
 http-server.http.port=8080
 discovery-server.enabled=true
 discovery.uri=http://localhost:8080
+
```

Comment:
remove these blank lines - one is enough

Reply:
Done